Sources/Controllers/2.0/OACustomPOIViewController.h

```diff
@@ -19,4 +19,6 @@
 
 - (instancetype)initWithFilter:(OAPOIUIFilter *)filter;
 
++ (void)updateSearchView:(BOOL)searchMode searchField:(UITextField * _Nonnull)searchField cancelButton:(UIButton * _Nonnull)cancelButton searchFieldRightConstraint:(NSLayoutConstraint * _Nonnull)searchFieldRightConstraint;
```

Comment:
What is the purpose of this method? I don't even see its implementation anywhere.